scripts/submit-codecov-data.sh

```diff
@@ -22,7 +22,8 @@ RETRY_DELAY=${RETRY_DELAY:-5}
 
 # Work around for temporary codecov API timing out
 for (( i=0; i<$MAX_ATTEMPTS; ++i)); do
-    codecov --root=/home/circleci/scalyr-agent-2/ --disable gcov --file coverage.xml
+    # NOTE: We pass --required flag to the binary since we also want it to rAeturn non-zero if upload fails
```

Comment:
typo

Reply:
Yep, thanks, fixed in 78401f5a22509541f639cd488a6071ccb0c54558.